src/utils/appDataClient.ts

```diff
@@ -140,6 +140,20 @@ export const getAccount = async (address?: string) => {
   return data
 }
 
+export const getAccountRewards = async (opts?: {
+  address?: string
+  numDaysBack?: number
+}) => {
+  Logger.breadcrumb('getAccountRewards', breadcrumbOpts)
+  const accountAddress = opts?.address || (await getAddress())
+  if (!accountAddress) return
+
+  const initialDate = new Date()
+  const endDate = new Date()
+  endDate.setDate(initialDate.getDate() - (opts?.numDaysBack || 1))
```

Comment:
this works, there are also some handy functions in the `date-fns` package for working with stuff like this

Reply:
yeah, that's what I would normally use, but I copied this from explorer 😄

Reply:
I'll use date-fns to be consistent with other date math in the app.